lib/cash-out/cash-out-tx.js

```diff
@@ -125,8 +124,8 @@ function monitorIncoming (settings, statuses, fromAge, toAge) {
   return fetchOpenTxs(statuses, fromAge, toAge)
     .then(txs => pEachSeries(txs, tx => processTxStatus(tx, settings)))
     .catch(err => {
-      if (err.code === SERIALIZATION_FAILURE_CODE) {
-        logger.warn(HARMLESS_DB_CONFLICT_ERROR)
+      if (err.code === dbConsts.errorCodes.SERIALIZATION_FAILURE) {
+        logger.warn(dbConsts.msgs.HARMLESS_DB_CONFLICT)
```

Comment:
Will this message be used more than once?

Reply:
Oh, I should have added it to the routes.js file, pushing the fix right now.